pkgs/shells/bash/ble.sh/default.nix

```diff
@@ -0,0 +1,66 @@
+{ lib, stdenvNoCC, fetchgit, git, curl, jq, nix-prefetch, runtimeShell }:
+
+stdenvNoCC.mkDerivation {
+  name = "ble.sh";
+  version = "0.3.3";
+
+  src = fetchgit {
+    url = "https://github.com/akinomyoga/ble.sh.git";
+    rev = "refs/tags/v0.3.3";
+    hash = "sha256-Gfo2S1t5Kdy+8TEDS4M5yhyRShvzQIljdE0MQK1CL+4=";
+    fetchSubmodules = true;
+    leaveDotGit = true;
+  };
+
+  nativeBuildInputs = [ git ];
+  buildInputs = [ curl jq nix-prefetch ];
+
+  doCheck = true;
+
+  installFlags = [ "INSDIR=$(out)/share" ];
+  postInstall = ''
+    mkdir -p "$out/bin"
+    cat <<SCRIPT >"$out/bin/ble.sh-share"
+    #!${runtimeShell}
+    # Run this script to find the ble.sh shared folder
+    # where all the shell scripts are living.
+    echo "$out/share/ble.sh"
+    SCRIPT
```

Comment:
```suggestion
    cat <<EOF >"$out/bin/ble.sh-share"
    #!${runtimeShell}
    # Run this script to find the ble.sh shared folder
    # where all the shell scripts are living.
    echo "$out/share/ble.sh"
    EOF
```

Reply:
It's not an End Of File. It's just an end mark for here-document, so I didn't use EOF on purpose.

Reply:
Please just use the default name. It makes it easier for everyone to see at a glance whats going on.

Reply:
Hmm, I don't think EOF is the default, as I have seen many people avoid EOF and use EOS or else. But it is actually EOF here, so I'm not so reluctant to switch to it.